README.md

```diff
@@ -285,6 +282,112 @@ func main() {
 }
 ```
 
+For use SQL to query to tarantool, call `Execute`. 
```

Comment:
I use Google Doc to verify syntax, it's really useful.

"To use SQL to query a tarantool instance, call `Execute`."

Reply:
Thank you for advice, I will use it onward! Fixed.